app/helpers/contact_helper.rb

```diff
@@ -0,0 +1,18 @@
+module ContactHelper
+  def split_first_and_last_name(full_name)
```

Comment:
Found this https://gist.github.com/sukima/491187

Reply:
Nice one. Thanks for sharing. I have added more cases.